src/libs/actions/User.js

```diff
@@ -8,8 +8,6 @@ function getBetas() {
     API.User_GetBetas().then((response) => {
         if (response.jsonCode === 200) {
             Onyx.set(ONYXKEYS.BETAS, response.betas);
-        } else {
-            console.error('Could not get betas', response);
```

Comment:
This was alarming for no reason if we can't get the betas then we try again because probably the authToken expired.